src/renderer/pages/app.js

```diff
@@ -1,39 +1,48 @@
 const colors = require('material-ui/styles/colors')
+const createGetter = require('fn-getter')
 const React = require('react')
 
 const darkBaseTheme = require('material-ui/styles/baseThemes/darkBaseTheme').default
-const lightBaseTheme = require('material-ui/styles/baseThemes/lightBaseTheme').default
 const getMuiTheme = require('material-ui/styles/getMuiTheme').default
 const MuiThemeProvider = require('material-ui/styles/MuiThemeProvider').default
 
 const Header = require('../components/header')
 
+// Perf optimization: Needed immediately, so do not lazy load it below
+const TorrentListPage = require('./torrent-list-page')
```

Comment:
You're right, but putting here at the top is still faster in my testing. It's better to do requires (which block the event loop) while async I/O is happening that we're waiting on before we can continue the startup process. In this case, we're still waiting on the `state.load()` call in `renderer/main.js` while this require happens. It's better to it while that's happening, then to do it later when there's no I/O happening.

Would be useful to have a proper waterfall graph for this to prove this rationale is correct, but this is what my testing seemed to point to.